packages/embark/src/lib/modules/blockchain_listener/index.js

```diff
@@ -60,8 +60,13 @@ class BlockchainListener {
     this.embark.registerAPICall(
       'get',
       '/embark-api/regular-txs',
-      (req, _res) => {
-        this.events.request(`regularTxs:${req.query.mode === 'on' ? 'start' : 'stop'}`);
+      (req, res) => {
+        this.events.request(`regularTxs:${req.query.mode === 'on' ? 'start' : 'stop'}`, (err, result) => {
```

Comment:
I'd prefer not doing this check here. In fact, I'd prefer checking for `on` or `off` and respond with 400 if neither of those values were provided. Either way, it's the way it was done before and I won't make a huge fuss.

Reply:
Updated with param validation.